packages/typography/src/Link/Link/Link.styles.ts

```diff
@@ -0,0 +1,25 @@
+import { css } from '@leafygreen-ui/emotion';
+
+import { anchorClassName } from '../shared.styles';
+
+export const arrowRightIconPersist = css`
+  transform: translate3d(3px, 0, 0);
+`;
+
+export const arrowRightIconHover = css`
+  opacity: 0;
+  transform: translate3d(-3px, 0, 0);
+  transition: all 100ms ease-in;
```

Comment:
_technically_ just transitioning `transform` and `opacity` right? This might be old info, but I _believe_ there's a slight perf. bump by specifying which properties are going to transitions (that may be out of dat now though)